Ensure that the local glance is used to build docs,
rather than the system version which might not even be installed.
This is reported upstream so may be fixed in Essex.
https://review.openstack.org/#change,1849

--- glance-2011.3.orig/doc/source/conf.py	2011-09-21 21:46:35.000000000 +0100
+++ glance-2011.3/doc/source/conf.py	2011-11-22 15:03:59.306468960 +0000
@@ -33,10 +33,10 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.append([os.path.abspath('../glance'),
-    os.path.abspath('..'),
-    os.path.abspath('../bin')
-    ])
+sys.path = [os.path.abspath('../../glance'),
+    os.path.abspath('../..'),
+    os.path.abspath('../../bin')
+    ] + sys.path
 
 # -- General configuration ---------------------------------------------------
 
